PWGJE/Tasks/jetfinderQA.cxx

```diff
@@ -820,7 +825,7 @@ struct JetFinderQATask {
   PROCESS_SWITCH(JetFinderQATask, processTriggeredData, "QA for charged jet trigger", false);
 
   void processTracks(soa::Filtered<JetCollisions>::iterator const& collision,
-                     soa::Filtered<JetTracks> const& tracks)
+                     soa::Filtered<soa::Join<JetTracks, aod::JTrackExtras>> const& tracks)
```

Comment:
the problem again here is these functions can no longer be used on derived data in this case since for now we dont intend to save JTrackExtras. You could make two functions, one with and one without it? or add the checks all to the trackEfficiency task. This would be even better